modules/ROOT/pages/_partials/afm-ref-schedule-interval.adoc

```diff
@@ -0,0 +1,41 @@
+[[scheduling-limits]]
+= Limits for Schedule Intervals by Location
+
+The minimum interval for schedules that run in private locations is always five minutes.
+
+The minimum interval for schedules that run in public locations depends on the number of schedule entitlements, or API monitor entitlements, that you have. If you have more than five, the limit is five minutes. Otherwise, it is fifteen minutes.
+
+The limit depends on the type of location, not on whether the API is public or private, because you can test a public API from a private location.
```

Comment:
```suggestion
The limit depends on the type of location, and not on whether the API is public or private because you can test a public API from a private location.
```

Reply:
Are you sure this is correct? The original looks correct to me. Could you please explain the edit?

Reply:
I checked the grammar in Google Docs and it said it is okay, so I'm going to keep it as-is if that's okay.